### PR TITLE
[[ Bug 17852 ]] Deal with conflicted values in segmented control PI editors

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: Toolset/palettes/inspector/editors/com.livecode.pi.textstyle.behavior.livecodescript [hunk 1]
```diff
@@ -4,13 +4,45 @@ on editorInitialize
...
+   put the keys of pConflicts into tKeys
+   repeat for each item tItem in pConflicts[line 1 of tKeys]
+      put true into tCommon[tItem]
+   end repeat
+   delete line 1 of tKeys
+   
+   local tCurrent
+   repeat for each line tKey in tKeys
+      repeat for each key tItem in tCommon
+         if tItem is among the items of pConflicts[tKeys] then
```
Comment (montegoulding) [inline Toolset/palettes/inspector/editors/com.livecode.pi.textstyle.behavior.livecodescript]: @livecodeali Is this meant to be `pConflicts[tKey]` here?

Comment (livecodeali) [inline Toolset/palettes/inspector/editors/com.livecode.pi.textstyle.behavior.livecodescript]: Yes, thanks!

**Comment by montegoulding** [comment]
@livecode-vulcan review ok a7a378c

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok a7a378ce3df4afd6cc0c743fd96c2e6501cb2656

**Pull request merged**